### PR TITLE
Update auto-generated documentation

### PR DESCRIPTION
**Pull request by snoopycrimecop**
Repository: openmicroscopy/ome-documentation
Already up-to-date.

Generated by build [OMERO-DEV-latest-docs-autogen#1556](https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/1556/).

----
--no-rebase

### CODE REVIEW

**Comment by joshmoore** [comment]
Think @snoopycrimecop gets a 🍖 for PR `2000`

**Review by mtbc — APPROVED** [review]

File: omero/sysadmins/unix/walkthrough/walkthrough_centos7.sh [hunk 1]
```diff
@@ -31,15 +31,15 @@ yum -y install openssl-devel bzip2-devel
 
 yum -y install ice-all-runtime
 
-pip install "zeroc-ice>3.5,<3.7"
+pip install https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.1.0/zeroc_ice-3.6.4-cp27-cp27mu-linux_x86_64.whl
```
Comment (mtbc) [inline omero/sysadmins/unix/walkthrough/walkthrough_centos7.sh:34]: I've been using the same one for new-learning.

Comment (joshmoore) [reply omero/sysadmins/unix/walkthrough/walkthrough_centos7.sh:34]: Oh definitely. More just a question of whether or not we're ready to expose users to this. So far we've kept this pretty internal, e.g. it's not the default in prod playbooks, etc.

cc: @manics @jburel

Comment (joshmoore) [reply omero/sysadmins/unix/walkthrough/walkthrough_centos7.sh:34]: If we do want to go this route, https://github.com/ome/zeroc-ice-py-centos7/releases/latest/download/zeroc_ice-3.6.4-cp27-cp27mu-linux_x86_64.whl might be one idea to be more stable. Another would be to introduce a redirect via .htaccess:

```
[jamoore@idr0-slot3 wheel]$ cat .htaccess
Redirect 301 /ice/wheel/ https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.1.0/zeroc_ice-3.6.4-cp27-cp27mu-linux_x86_64.whl
```

**Pull request merged**